kalite/distributed/features/superuser_create.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Create superuser from the in browser modal
+
+    Scenario: Superuser already exists
+        Given there is superuser 
+        And I am on the homepage
+        Then there should be no modal displayed
+
+    Scenario: Superuser does not exist yet
+        Given there is no superuser
+        And I am on the homepage
+        Then I should see a modal
+        And I will not be able to dismiss the modal until a superuser is created
+
+    Scenario: Create superuser with empty username
+        Given the username is empty
+        When I click the create button
+        Then the username border will turn red
+        And the modal won't dismiss
+
+    Scenario: Create superuser with string longer than 40
+        Given I enter a username longer than 40 letters
+        When I click the create button
+        Then the modal won't dismiss
+
+    Scenario: Create superuser with empty password
```

Comment:
Given that the form has its own validation, I think it should be sufficient to only have one test with bad input. Every bad input scenario ends in the same result (`Then the modal won't dismiss`), so in my opinion not much is gained from testing every possible error condition vs the cost of running a lot more relatively slow Selenium tests.